dashboard/app/models/pd/enrollment.rb

```diff
@@ -169,24 +161,17 @@ def self.filter_for_survey_completion(enrollments, select_completed = true)
 
     # Local summer, CSP Workshop for Returning Teachers, or CSF Intro after 5/8/2020 will use Foorm for survey completion.
     # CSF Deep Dive after 9/1 also uses Foorm. CSF District workshops will always use Foorm
-    foorm_enrollments, other_enrollments = enrollments.partition do |enrollment|
```

Comment:
TIL about partition! Very cool –– and also a pleasure to be able to delete this logic